### PR TITLE
LOOP-3064 Omnipod First Run

### PR DESCRIPTION
**Pull request by ps2**
https://tidepool.atlassian.net/browse/LOOP-3064

### CODE REVIEW

File: DashKitUI/PumpManager/DashHUDProvider.swift [hunk 1]
```diff
@@ -55,8 +55,9 @@ internal class DashHUDProvider: NSObject, HUDProvider {
         return reservoirView
     }
 
-    public func didTapOnHUDView(_ view: BaseHUDView) -> HUDTapAction? {
-        return HUDTapAction.presentViewController(pumpManager.settingsViewController(bluetoothProvider: bluetoothProvider, colorPalette: colorPalette))
+    public func didTapOnHUDView(_ view: BaseHUDView, allowDebugFeatures: Bool) -> HUDTapAction? {
```
Comment (nhamming) [inline DashKitUI/PumpManager/DashHUDProvider.swift:58]: [nit] should `allowDebugFeatures` default to `false`?

File: DashKitUI/PumpManager/DashPumpManager+UI.swift [hunk 2]
```diff
@@ -18,17 +18,17 @@ extension DashPumpManager: PumpManagerUI {
         return UIImage(named: "Onboarding", in: Bundle(for: DashSettingsViewModel.self), compatibleWith: nil)
     }
 
-    public static func setupViewController(initialSettings settings: PumpManagerSetupSettings, bluetoothProvider: BluetoothProvider, colorPalette: LoopUIColorPalette) -> SetupUIResult<PumpManagerViewController, PumpManagerUI> {
-        let vc = DashUICoordinator(colorPalette: colorPalette, pumpManagerType: self, initialSettings: settings)
+    public static func setupViewController(initialSettings settings: PumpManagerSetupSettings, bluetoothProvider: BluetoothProvider, colorPalette: LoopUIColorPalette, allowDebugFeatures: Bool) -> SetupUIResult<PumpManagerViewController, PumpManagerUI> {
```
Comment (nhamming) [inline DashKitUI/PumpManager/DashPumpManager+UI.swift:21]: [nit] should `allowDebugFeatures` default to `false`?

If you agree, same comment for others.

File: DashKitUI/Views/PodSetupView.swift [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    private var title: some View {
+        Text("Pod Setup")
+            .font(.largeTitle)
+            .bold()
+            .padding(.vertical)
+    }
+    
+    @ViewBuilder
+    private var bodyText: some View {
+        Text("You will now begin the process of configuring your reminders, filling your Pod with insulin, pairing to to your device and placing it on your body.")
```
Comment (nhamming) [inline DashKitUI/Views/PodSetupView.swift]: localize?

File: DashKitUI/Views/PodSetupView.swift [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+            Divider()
+            bodyText
+                .foregroundColor(.secondary)
+                .padding(.top)
+        }
+    }
+
+    @ViewBuilder
+    private var title: some View {
+        Text("Pod Setup")
```
Comment (nhamming) [inline DashKitUI/Views/PodSetupView.swift]: Localize?

File: DashKitUI/Views/PodSetupView.swift [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+        Button("Continue", action: nextAction)
+            .buttonStyle(ActionButtonStyle())
+    }
+    
+    private var skipOnboardingAlert: Alert {
+        Alert(title: Text("Skip Omnipod Onboarding?"),
+              message: Text("Are you sure you want to skip Omnipod Onboarding?"),
+              primaryButton: .cancel(),
+              secondaryButton: .destructive(Text("Yes"), action: skipOnboarding))
+    }
```
Comment (nhamming) [inline DashKitUI/Views/PodSetupView.swift]: Localize?

**Review by nhamming — APPROVED** [review]
Just missing localization and nits. LGTM

File: DashKitUI/PumpManager/DashHUDProvider.swift (hunk 1 above)
Comment (ps2) [reply DashKitUI/PumpManager/DashHUDProvider.swift:58]: `didTapOnHUDView` is a protocol method.

File: DashKitUI/Views/PodSetupView.swift (hunk 5 above)
Comment (ps2) [reply DashKitUI/Views/PodSetupView.swift]: Probably shouldn't put debug UI into localization.

Comment (ps2) [reply DashKitUI/Views/PodSetupView.swift]: Oh, I see, the "Continue" button. Thanks.

**Pull request merged**